packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer.spec.ts

```diff
@@ -10,4 +31,206 @@ describe('RichTextViewer', () => {
     it('should export its tag', () => {
         expect(richTextViewerTag).toBe('nimble-rich-text-viewer');
     });
+
+    it('should reflect value of the "markdown" to the internal control', async () => {
```

Comment:
I don't think this test is covering what you intend. Usually we write these "should reflect...to the internal control" tests to ensure that setting the attribute will cause the property to update and vice versa. But this is checking that setting the attribute causes the attribute to update, which is just basic behavior of any HTML element.

I believe we decided explicitly NOT to support a `markdown` attribute on the element (at least that's what's implemented). So some valid tests would be:
1. set the markdown attribute and ensure the markdown property isn't modified
2. set the markdown property and ensure there is no markdown attribute
3. set the markdown property and ensure that getting the markdown property returns the same value

(I'm using "property" loosely here to refer to the `element.markdown` accessors)

Reply:
Thanks for pointing this out. I have updated the tests accordingly and added these valid test cases.